lib/libpcp/src/pcp_utils.h

```diff
@@ -219,8 +219,10 @@ static inline void createNonce(struct pcp_nonce *nonce_field)
     int i;
     for (i = 2; i >= 0; --i)
 #ifdef WIN32
+		// DISCUSSME: what should we do here?
         nonce_field->n[i]=htonl (rand());
 #else  //WIN32
+		// DISCUSSME: what should we do here?
         nonce_field->n[i]=htonl(random());
 #endif //WIN32
```

Comment:
Nothing. Do nothing.

Reply:
So I shouldn't integrate our third-party libraries (specifically, libpcp and liblua) into this PR? Should they continue to use `rand()`/`srand()`, then?

Reply:
Never mind, I just read your other comment about not messing with third-party libraries, so I'll leave those untouched. I figured it was worth asking, though.

Reply:
Sorry, I should have been more descriptive here. Made the assumption that you would have read my earlier comment first for the reason and just meant to tag this change something to be ignored.

Reply:
It's cool, and agreed that tinkering with third-party library code is generally a bad idea, for the reasons you mentioned.

In this particular case, I wasn't sure if the game would run properly if Lua (no idea what libpcp even does) was still using `rand()`/`srand()`.